### ISSUE TITLE
RD porn section broken again

### ISSUE THREAD

**Issue by TrillaG** [team-crew/team-crew.github.io#151]
Recent videos just all have links to images, no videos. The rest won't load at all.

**Comment by fade2gray** [comment]
If you want support from the TC devs, see...

![TC_GitHub](https://github.com/team-crew/team-crew.github.io/assets/1332009/f7680eea-9bd2-4fea-aae4-04517fcd0eb1)

**Comment by TrillaG** [comment]
You say this everytime, but multiple people have tried telling you that link expires and is broken so there's no way to do that.

**Comment by fade2gray** [comment]
How many people? Oh, yeah - one, Zippy457. Not exactly the right way to get someone to enquire on your behalf, is it?

Good luck.

**Comment by classymouse** [comment]
@TrillaG, we are aware of the problem and working on a fix. Maybe next time give us a shout in the Telegram group?

**Comment by TrillaG** [comment]
Well, make it 2 buddy, cause that shit doesn't work. Get mad at me about it, I'm sure that makes sense to you lol clown @fade2gray

**Comment by TrillaG** [comment]
The link doesn't work I tried @classymouse it worked for me the very first time but never since, seems like it expires or something Idk... I know I didn't get banned or anything like that.

**Comment by fade2gray** [comment]
@TrillaG - Guessing you'll have a long wait the next time you need advice. Too bad.

**Comment by TrillaG** [comment]
Well it's a good thing I will never need advice from you. Are you really power tripping on a GitHub issues post? How delusional are you? Get a grip dude. You're mad for literally no reason.. re-evaluate.

**Comment by fade2gray** [comment]
Classy only arrived because of the message I sent on your behalf, White Hat only pops in every 6 months or so, you can't get into Telegram - yeah, too bad. Consider yourself muted, you ingrate.

**Comment by TrillaG** [comment]
I can get into Telegram fine, your link is not working as others and I have tried telling you. But that's a touchy subject for you, and I definitely don't wanna get you too upset, please don't use your ultimate powers on me. You fucking pussy.

**Comment by TrillaG** [comment]
Just relay the message errand boy and let the big boys solve the problems.

**Comment by classymouse** [comment]
@TrillaG Just FYI, i am a girl, tiny diff. We are very busy and we ARE taking comments seriously, but with respect, as i showed you and as i would like you to show to me. Is the problem that you cannot get into the Telegram group?

**Comment by TrillaG** [comment]
Yeah none of that was directed at you sorry if you took offence @classymouse I appreciate it I know you're busy but yeah the reason he got mad is because the link that he provides for the Telegram does not work as another user has also stated. It worked for me once the first time but never since (and I know I wasn't banned) so yeah I can't access the Telegram to make you aware of any problems, in this case with the Adult section of Real Debrid links.

**Comment by classymouse** [comment]
Yes, we are aware of the problems in the adul;t section. Tonight (in whichever timezone you reside it may differ) we hope to release a new version which solves a lot of the problems. We are working hard on all ends, code and scrapers to fix things and yes, @fade2gray was factually right, more and more "free websites" disappear and we are working hard to fix that but it is a struggle.

**Comment by TrillaG** [comment]
Okay, sounds good. Thank you for letting me know, I appreciate your work. And just to let you know, the exact message for the Telegram link is that it "is invalid or has expired", so Idk if maybe we can try getting an updated link on here for the Telegram group for any future issues.

**Comment by classymouse** [comment]
The links expire because of a lot of reasons i have no influence of. If you want a new link, look me up on Telegram, send me a PM and i hook you up with a new link ;-)
